site/content/en/docs/handbook/pushing.md

```diff
@@ -20,7 +20,7 @@ Here is a comparison table to help you choose:
 |---	|---	|---	|---	|---	|
 |  [docker-env command](/docs/handbook/pushing/#1pushing-directly-to-the-in-cluster-docker-daemon-docker-env)	|   only docker	|  good 	|
 |  [podman-env command](/docs/handbook/pushing/#3-pushing-directly-to-in-cluster-crio-podman-env)	|   only cri-o |  good 	|
-|  [cache add command](/pushing/#2-push-images-using-cache-command) 	|  all 	|  ok 	|
+|  [cache add command](/docs/commands/cache/#minikube-cache-add) 	|  all 	|  ok 	|
```

Comment:
same as these, change them to "Ref" links

Reply:
Incorporated the comments